components/has/overlays/dev/apiexport-patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: has
+spec:
+  permissionClaims:
+  - group: ""
+    resource: secrets
+  - group: ""
+    resource: configmaps
+  - identityHash: gitopsrvc-appstudio-shared
+    group: "appstudio.redhat.com"
+    resource: "environments"
+  - identityHash: gitopsrvc-appstudio-shared
+    group: "appstudio.redhat.com"
+    resource: "applicationsnapshotenvironmentbindings"
```

Comment:
I would rather keep the permissionClaims together with the rest of the APIExport in the API repository and patch only the actual identityHash here. 
I know that it would mean that we need to rely on the position in the list of permission claims to be able to set the correct identity hash, but it's still better than replicating the whole permissionClaims section here.

Reply:
For debugging purposes it's easier to have it in one file. Validation if API exporter points to right identity hash could be complicated. Also this format gives me way to apply this file directly to create placeholder IdentityHash when apiexport does not exists yet but we need it's identityHash

Reply:
> For debugging purposes it's easier to have it in one file. Validation if API exporter points to right identity hash could be complicated. 

I fully understand that, but this would mean that we need to keep the permissionClaims in sync with the original APIExport, right? And have a validation that checks if there is any update in the original permissionClaims which wasn't reflected here. This can be tricky as well.

> Also this format gives me way to apply this file directly to create placeholder IdentityHash when apiexport does not exists yet but we need it's identityHash

But wouldn't it be easier to just create a set of empty APIExports without any APIResourceSchema or permissionClaims? You technically don't need that, right?

Reply:
Wondering too if we should wait on the discussions around using a shared APIExport for HAS + gitops-appstudio-shared? If we do move to that

Reply:
We need to have permissionClaims so that we can identify to which APIExport it should point via IdentityHash.